package.json

```diff
@@ -51,52 +51,52 @@
   "devDependencies": {
     "@iarna/toml": "^2.2.5",
     "@istanbuljs/nyc-config-typescript": "^1.0.2",
-    "@microsoft/api-extractor": "^7.33.7",
+    "@microsoft/api-extractor": "^7.34.4",
     "@microsoft/tsdoc-config": "^0.16.2",
-    "@mongodb-js/zstd": "^1.0.0",
-    "@types/chai": "^4.3.3",
+    "@mongodb-js/zstd": "^1.1.0",
+    "@types/chai": "^4.3.4",
     "@types/chai-subset": "^1.3.3",
-    "@types/express": "^4.17.14",
+    "@types/express": "^4.17.17",
     "@types/kerberos": "^1.1.1",
-    "@types/mocha": "^10.0.0",
-    "@types/node": "^18.11.0",
+    "@types/mocha": "^10.0.1",
+    "@types/node": "^18.15.1",
     "@types/saslprep": "^1.0.1",
-    "@types/semver": "^7.3.12",
+    "@types/semver": "^7.3.13",
     "@types/sinon": "^10.0.13",
-    "@types/sinon-chai": "^3.2.8",
+    "@types/sinon-chai": "^3.2.9",
     "@types/whatwg-url": "^11.0.0",
-    "@typescript-eslint/eslint-plugin": "^5.48.0",
-    "@typescript-eslint/parser": "^5.48.0",
-    "chai": "^4.3.6",
+    "@typescript-eslint/eslint-plugin": "^5.54.1",
+    "@typescript-eslint/parser": "^5.54.1",
+    "chai": "^4.3.7",
     "chai-subset": "^1.6.0",
     "chalk": "^4.1.2",
-    "eslint": "^8.31.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-import": "^2.26.0",
+    "eslint": "^8.36.0",
+    "eslint-config-prettier": "^8.7.0",
+    "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-simple-import-sort": "^8.0.0",
+    "eslint-plugin-simple-import-sort": "^10.0.0",
     "eslint-plugin-tsdoc": "^0.2.17",
     "express": "^4.18.2",
     "js-yaml": "^4.1.0",
-    "mocha": "^9.2.2",
+    "mocha": "^10.2.0",
     "mocha-sinon": "^2.1.2",
-    "mongodb-legacy": "^4.0.0",
+    "mongodb-legacy": "^5.0.0",
     "nyc": "^15.1.0",
-    "prettier": "^2.7.1",
-    "rimraf": "^3.0.2",
+    "prettier": "^2.8.4",
+    "rimraf": "^4.4.0",
```

Comment:
From the [4.x major changelog of rimraf](https://github.com/isaacs/rimraf/blob/main/CHANGELOG.md):
```
Remove glob dependency entirely. This library now only accepts actual file and folder names to delete.
```

Our build:tsd command is:
```
"build:dts": "npm run build:ts && api-extractor run && rimraf 'lib/**/*.d.ts*'",
```
So that's just gunna rm `lib` maybe? 😳😅

Reply:
Nvm, as you pointed out, it was added back in 4.2 
https://github.com/isaacs/rimraf/blob/main/CHANGELOG.md#v42